### PR TITLE
mklfpk and tbb were removed; scripts to download mklfpk and tbb added; azyre-pipeline changed

### PR DESCRIPTION
**Pull request by vmeshche**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -19,12 +19,15 @@ jobs:
...
     maxParallel: 2
   steps:
-  - script: git lfs fetch && git lfs checkout && du -hs externals/mklfpk/lnx/lib/intel64/*
-  - script: sudo apt-get update
   - script: sudo apt-get install gcc-multilib g++-multilib openjdk-8-jdk
-  - script: ls -al /usr/lib/jvm/java-8-openjdk-amd64 ; ls -al /usr/lib/jvm/java-8-openjdk-amd64/include
+    displayName: 'apt-get'
   - script: |
-      export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64; export PATH=$JAVA_HOME/bin:$PATH; export CPATH=$JAVA_HOME/include:$JAVA_HOME/include/linux:$CPATH && make daal -j4 PLAT=$(platform.type) REQCPU="avx2" COMPILER=gnu
+      scripts/mklfpk.sh intel64
```
Comment (napetrov) [inline azure-pipelines.yml]: do we copy intel64 bit version even for 32 bit build?

File: scripts/mklfpk.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+while [ 1 ] ; do
+    if [ "$1" = "--help" ] ; then
+        echo "Usage: $0 [ia32|intel64]"
+        echo "Usage example: $0 intel64"
+        exit 1
+    elif [ "${1}" = "ia32" ] ; then
+        MKLFPK_ARCH=ia32
+    elif [ "${1}" = "intel64" ] ; then
+        MKLFPK_ARCH=intel64
```
Comment (napetrov) [inline scripts/mklfpk.sh]: why we need param if it's not used?

File: scripts/mklfpk.sh [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+# use, modify, copy, publish, distribute,  disclose or transmit this software or
+# the related documents without Intel's prior written permission.
+#
+# This software and the related documents  are provided as  is,  with no express
+# or implied  warranties,  other  than those  that are  expressly stated  in the
+# License.
+#===============================================================================
+
+
+MKLFPK_URL_ROOT="https://github.com/vmeshche/daal/releases/download/2019_u3/"
```
Comment (napetrov) [inline scripts/mklfpk.sh]: Let's switch to product repo and upload mklfp to 2019_u4 release ?

File: azure-pipelines.yml (hunk 1 above)
Comment (vmeshche) [reply azure-pipelines.yml]: fixed

File: scripts/mklfpk.sh (hunk 2 above)
Comment (vmeshche) [reply scripts/mklfpk.sh]: Fixed

File: scripts/mklfpk.sh (hunk 3 above)
Comment (vmeshche) [reply scripts/mklfpk.sh]: PLease upload release. I will change url

**Comment by vmeshche** [comment]
changes will be reimplemented :)

File: scripts/mklfpk.sh (hunk 2 above)
Comment (vmeshche) [reply scripts/mklfpk.sh]: params section were saved. parameter passing were organized.

File: azure-pipelines.yml (hunk 1 above)
Comment (vmeshche) [reply azure-pipelines.yml]: params section were saved. parameter passing were organized.

**Pull request merged**